src/xpk/commands/info.py

```diff
@@ -162,7 +162,7 @@ def get_flavors_resources_reservations(
   reservations[cq_name] = {}
   for flavor_name, flavor_resources_reservation_list in flavors_res.items():
     for resource in flavor_resources_reservation_list:
-      reservations[cq_name][f'{flavor_name}:{resource["name"]}'] = resource[
+      reservations[cq_name][f'{flavor_name}:{resource.name}'] = resource[
```

Comment:
This is not connected with adding batch flags.  As it is minor change it may be here, but please do not unrelated changes to PR

Reply:
Just fix for linter.